Leanda.Categories/Domain/Events/EntityCategoriesCreated.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Leanda.Categories.Domain.ValueObjects;
+using Sds.CqrsLite.Events;
+using System;
+using System.Collections.Generic;
+
+namespace Leanda.Categories.Domain.Events
+{
+    public class EntityCategoriesCreated : IUserEvent
```

Comment:
same here: interface

Reply:
In this class we have a constructor. I can't use an interface.